packages/tools/devtools/devtools-view/package.json

```diff
@@ -119,6 +121,7 @@
 		"eslint-plugin-react-hooks": "~4.6.0",
 		"globby": "^13.2.2",
 		"jest": "^29.6.2",
+		"jest-axe": "^8.0.0",
```

Comment:
jest-axe's docs suggest a usage pattern I don't see here, so are we really leveraging it?

Reply:
Ah, I had tried it but didn't use it. Will remove.